.github/workflows/django.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
time to change the admin repo.. @goapunk

Reply:
@m4ra I agree, not sure if we have the capacities yet though?